src/PhpSpreadsheet/Style/NumberFormat/DateFormatter.php

```diff
@@ -112,23 +160,23 @@ public static function format($value, string $format): string
         // escape any quoted characters so that DateTime format() will render them correctly
         /** @var callable */
         $callback = ['self', 'escapeQuotesCallback'];
-        $format = preg_replace_callback('/"(.*)"/U', $callback, $format);
```

Comment:
While I'm reminded: callbacks in the format `['self', 'escapeQuotesCallback']` are being deprecated in PHP 8.2; the valid replacement format is `[self::class, 'escapeQuotesCallback']`

Reply:
Also worth noting:
```php
$format = preg_replace_callback('/"(.*)"/U', $callback, $format);
```
can return null or a string; and phpstan may subsequently complain when that value is passed as an argument to another function. Rather than doing `anotherFunction($format ?? '')`, it's probably better to enforce the string earlier with an explicit cast of the return from `preg_replace_callback()`
```php
$format = (string) preg_replace_callback('/"(.*)"/U', $callback, $format);
```

Reply:
Changes applied in latest commit. It looks like there are only 2 other places in PhpSpreadsheet where 'self' is used.